README.md

```diff
@@ -108,24 +119,127 @@ The compiled workflow can be executed via the DNAnexus command line client or we
     ```
-    $ cwl-upgrader my-workflow.cwl
+    $ pip install cwl-upgrader
+    $ cwl-upgrader my-workflow.cwl [subworkflow1.cwl subworkflow2.cwl ...]
     ```
-2. Pack it:
+2. Install `sbpack` package and use the `cwlpack` command on the top-level workflow file to build the "packed" one:
     ```
+    $ pip install https://github.com/rabix/sbpack.git`
     $ cwlpack --add-ids --json my-workflow.cwl > my-workflow.cwl.json
     ```
 3. De-localize all local paths referenced in your CWL: if your CWL specifies a local path, e.g. a schema or a default value for a `file`-type input, you need to upload those files to a DNAnexus project and then replace the local path in your CWL with a DNAnexus URI, e.g. `dx://project-XXX:file-YYY`.
 
+### Compile and run workflow
+The same `bam_chrom_counter` workflow as the example WDL above is now written in CWL v1.0:
+```
+cwlVersion: v1.0
+$graph:
+- id: bam_chrom_counter
+  class: Workflow
+  requirements:
+  - class: ScatterFeatureRequirement
+  inputs:
+  - id: bam
+    type: File
+  outputs:
+  - id: bai
+    type: File
+    outputSource: slice_bam/bai
+  - id: count
+    type: int[]
+    outputSource: count_bam/count
+  steps:
+  - id: slice_bam
+    run: "#slice_bam"
+    in:
+      bam: bam
+    out: [bai, slices]
+  - id: count_bam
+    run: "#count_bam"
+    scatter: bam
+    in:
+      bam: slice_bam/slices
+    out: [count]
+- id: slice_bam
+  class: CommandLineTool
+  inputs:
+  - id: bam
+    type: File
+    inputBinding:
+      position: 1
+  - id: num_chrom
+    default: 22
+    type: int
+    inputBinding:
+      position: 2
+  outputs:
+  - id: bai
+    type: File
+    outputBinding:
+      glob: $(inputs.bam).bai
+  - id: slices
+    type: File[]
+    outputBinding:
+      glob: "*.bam"
+  requirements:
+  - class: InlineJavascriptRequirement
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: "quay.io/biocontainers/samtools:1.12--hd5e65b6_0"
+  - class: InitialWorkDirRequirement
+    listing:
+    - entryname: slice_bam.sh
+      entry: |-
+        set -ex
+        samtools index $1
+        mkdir slices/
+        for i in `seq $2`; do
+            samtools view -b $1 -o slices/$i.bam $i
+        done
+  baseCommand: ["sh", "slice_bam.sh"]
+- id: count_bam
+  class: CommandLineTool
+  requirements:
+  - class: InlineJavascriptRequirement
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: "quay.io/biocontainers/samtools:1.12--hd5e65b6_0"
+  inputs:
+  - id: bam
+    type: File
+    inputBinding:
+      position: 1
+  baseCommand: ["samtools", "view", "-c"]
+  outputs:
+  - id: count
+    type: int
+    outputBinding:
+      glob: stdout
+      loadContents: true
+      outputEval: "$(parseInt(self[0].contents))"
+  stdout: stdout
+```
+Once it is upgraded and packed into `bam_chrom_counter.cwl.json` as suggested above, we can compile it to the DNAnexus platform and run it.
```

Comment:
How about already pasting here the 1.2 version so that the user can start quicker and skip the upgrade version?